tests/torch/ptq/test_calculation_quantizer_params.py

```diff
@@ -291,12 +291,12 @@ def calculate_fq_params(model, input_data):
     conv2_w = model.conv2.weight
     conv2_w_stats = calculate_statistics(conv2_w, QuantizationMode.SYMMETRIC, QuantizerGroup.WEIGHTS)
     return {
-        "FakeQuantize_6": conv1_stats,
-        "relu/FakeQuantize": bn1_stats,
-        "avg_pool/FakeQuantize": avg_pool_stats,
-        "bn1/FakeQuantize": conv2_stats,
-        "conv1/pre_ops.0/op/FakeQuantize": conv1_w_stats,
-        "conv2/pre_ops.0/op/FakeQuantize": conv2_w_stats,
+        "/FakeQuantize": conv1_stats,
+        "/relu/FakeQuantize": bn1_stats,
+        "/avg_pool/FakeQuantize": avg_pool_stats,
+        "/bn1/FakeQuantize": conv2_stats,
+        "/conv1/pre_ops.0/op/FakeQuantize": conv1_w_stats,
+        "/conv2/pre_ops.0/op/FakeQuantize": conv2_w_stats,
     }
```

Comment:
Looks like the name building algorithm in ONNX tracing code uses the same `Scope`-like mechanism now like we do to disambiguate operation names. This is probably triggered by a `torch.nn.Module.__call__`, and the top-level version of this `__call__` was missing after the NNCFNetwork refactor. These are the reference names of the ONNX nodes after export, and the new format is correct for the non-NNCF case, so the fix in this PR also fixes the ONNX node name building. The wrong format ended up being in the tests because this test had been added after the NNCFNetwork refactor.